CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### New features
 
+* Support background filesystem monitoring via watchman triggers enabled with
```

Comment:
suggestion: Can you also add a note to `docs/config.md` so that it appears in the doc website?

Reply:
Done